src/expr/impl/src/udf/quickjs.rs

```diff
@@ -85,37 +114,43 @@ struct QuickJsFunction {
 #[async_trait::async_trait]
 impl UdfImpl for QuickJsFunction {
     async fn call(&self, input: &RecordBatch) -> Result<RecordBatch> {
-        self.runtime.call(&self.identifier, input)
+        // TODO(eric): if not batched, call JS function row by row. Otherwise, one row failure will fail the entire chunk.
```

Comment:
I don't quite get this line. IIUC, the current status is to call JS function row by row if not batched, and failure of one row will fail the chunk, right? What's the todo mean?

Reply:
I think it's exactly the issue described in https://github.com/risingwavelabs/risingwave/issues/20473?